trunner/config.py

```diff
@@ -214,6 +220,23 @@ def parse_exec(self, config: Config) -> None:
 
         config['exec'] = shlex.split(exec_cmd)
 
+    def parse_prompt(self, config: Config) -> None:
+        prompt = config.get('prompt')
+        if not prompt:
+            return
+
+    def parse_psh(self, config: Config) -> None:
+        psh = config.get('psh')
+        if not psh:
+            return
+        psh = str(psh).lower()
+        if psh == 'true':
+            config['psh'] = True
+        elif psh == 'false':
+            config['psh'] = False
+        else:
+            raise ParserError(f'psh value of {psh} is not "True" or "False"')
```

Comment:
There is no need for pasing true/false, yaml package do it for us. Look how `ignore` keyword is parsed.